src/meta/src/dashboard/mod.rs

```diff
@@ -233,6 +335,12 @@ impl DashboardService {
                 get(prometheus::list_prometheus_actor_back_pressure),
             )
             .route("/monitor/await_tree/:worker_id", get(dump_await_tree))
+            .route("/monitor/dump_heap_profile/:worker_id", get(heap_profile))
```

Comment:
Should this be a POST interface?

Reply:
Good catch. Seems also apply to `await-tree`.

Reply:
We can open another pr to fix this along with await-tree